managed/models/database.go

```diff
@@ -1058,36 +1061,64 @@ func setupFixture1(q *reform.Querier, username, password string) error {
 	if _, err = CreateNodeExporter(q, PMMServerAgentID, nil, false, []string{}, nil, ""); err != nil {
 		return err
 	}
+	address, port, err := preparePGAddress(params.Address)
+	if err != nil {
+		return err
+	}
+	if params.Address != DefaultPostgreSQLAddr {
+		if node, err = CreateNode(q, RemoteNodeType, &CreateNodeParams{
+			NodeName: "pmm-server-db",
```

Comment:
It's better to keep strings in constants as it's done for `PMMServerPostgreSQLServiceName`.

Reply:
constant added